### PR TITLE
omero: Minor small tweaks to the versioning

### PR DESCRIPTION
**Pull request by ghost**
Minor fixes to the version information to fix typos/inconsistencies, and also revise 32-bit support in 5.2.

### CODE REVIEW

File: omero/sysadmins/version-requirements.txt [hunk 1]
```diff
@@ -27,8 +27,8 @@ Operating systems
 
 * Microsoft Windows
 
-  * Proposal: [5.1] 7 (client) recommended / Server 2008R2 (server)
-    recommended; XP/Server 2003/Vista dropped
+  * Proposal: [5.1] XP/Server 2003/Vista dropped, 7 (client)
+    recommended / Server 2008R2 (server) recommended
   * Proposal: [5.2] 7 / Server 2008R2 supported; 8 (client)
```
Comment (hflynn) [inline omero/sysadmins/version-requirements.txt]: Looks to me like this have a comma rather than a semi-colon to be consistent with the rest

**Comment by hflynn** [comment]
I think this is fine, I'm not even really convinced it is worth pushing another commit over a comma!

**Comment by hflynn** [comment]
Shall I just merge and release it for you?

**Comment by hflynn** [comment]
--no-rebase

**Comment by rleigh-dundee** [comment]
Pushed a fix anyway!

**Comment by hflynn** [comment]
Great, will merge once everything is green

**Comment by rleigh-dundee** [comment]
Thanks

**Pull request merged**